src/tests/fableconfig.json

```diff
@@ -13,7 +13,7 @@
             "projFile": ["Main/project.fsx"],
             "ecma": "es2015",
             "module": "commonjs",
-            "coreLib": "../../build/fable-core/umd",
+            "coreLib": "../../build/fable-core",
```

Comment:
Is this necessary to pass the tests? The default fable-core distribution now is in ES2015 module format, which is not understandable by node. That's why the tests use the UMD (universal module definition) distribution.